openfisca_us/variables/finance/tax/outputs.py

```diff
@@ -1244,18 +1253,71 @@ class ymod(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
-    documentation = (
-        """search taxcalc/calcfunctions.py for how calculated and used"""
-    )
+    label = "OASDI benefit tax variable"
+    documentation = "Variable that is used in OASDI benefit taxation logic"
+    unit = "currency-USD"
+
+    def formula(tax_unit, period, parameters):
+        ymod2 = (
+            tax_unit("filer_e00400", period)
+            + (0.5 * tax_unit("e02400", period))
+            - tax_unit("c02900", period)
+        )
+        ymod3 = add(
+            tax_unit, period, "filer_e03210", "filer_e03230", "filer_e03240"
+        )
+        return tax_unit("ymod1", period) + ymod2 + ymod3
 
 
 class ymod1(Variable):
     value_type = float
     entity = TaxUnit
     definition_period = YEAR
-    documentation = (
-        """search taxcalc/calcfunctions.py for how calculated and used"""
-    )
+    label = "AGI increase"
+    documentation = "Variable that is included in AGI"
+
+    def formula(tax_unit, period, parameters):
+        direct_inputs = add(
+            tax_unit,
+            period,
+            *[
+                "filer_" + variable
+                for variable in (
+                    "e00200",
```

Comment:
Make this something like `DIRECT_INPUT_COLS` (same with investment below) to reduce nesting

Reply:
Thanks, added